functionary/ui/templates/base.html

```diff
@@ -32,6 +23,11 @@
         <link rel="stylesheet"
               href="{% static 'css/functionary.css' %}"
               type="text/css"/>
+        {# Load the bootstrap libraries #}
+        {% bootstrap_css %}
+        {% bootstrap_javascript %}
```

Comment:
This ends up pulling the bootstrap JS and css files from a CDN. I think for the purpose of version control, we might want to just download the bootstrap files ourselves and load it like the rest of our static assets